content/en/docs/reference/access-authn-authz/validating-admission-policy.md

```diff
@@ -274,6 +190,8 @@ For the use cases require parameter configuration, we recommend to add a param c
   message: "params missing but required to bind to this policy"
 ```
 
+##### Optional P=parameters
```

Comment:
```suggestion
##### Optional parameters
```

Reply:
fixed this typo and adjusted the headings